com.github.kagami.boram.yml

```
@@ -0,0 +1,241 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=home
+  - --filesystem=/mnt
+  - --filesystem=/media
```

```suggestion
  - --filesystem=/media
  - --filesystem=/run/media
```

the latter is the standard location, the former is for some popular distro that break the stanard. `/mnt` isn't